### ISSUE TITLE
rxjs install issues

### ISSUE THREAD

**Issue by michielbdejong** [inrupt/generator-solid-react#164]
related to https://github.com/ReactiveX/rxjs/issues/4456#issuecomment-452164765 - rxjs install hangs during `yo @inrupt/solid-react`, to fix it, remove package-lock.json:

```sh
   create test/__mocks__/useLdflex.js
   create yarn.lock
Installing dependencies...
⸨            ░░░░░░⸩ ⠸ extract:rxjs: sill extract rxjs@6.3.3 extracted to /Users/michiel/gh/michielbdejong/solid-app-from-generator/node_modules/.staging/rxjs-38e51734 (9025ms)
⸨            ░░░░░░⸩ ⠸ extract:rxjs: sill extract rxjs@6.3.3 extracted to /Users/michiel/gh/michielbdejong/solid-app-from-generator/node_modules/.staging/rxjs-38e51734 (9025ms)
⸨            ░░░░░░⸩ ⠸ extract:rxjs: sill extract rxjs@6.3.3 extracted to /Users/michiel/gh/michielbdejong/solid-app-from-generator/node_modules/.staging/rxjs-38e51734 (9025ms)

~/gh/michielbdejong/solid-app-from-generator $ 
~/gh/michielbdejong/solid-app-from-generator $ npm install

~/gh/michielbdejong/solid-app-from-generator $ npm install rxjs

~/gh/michielbdejong/solid-app-from-generator $ rm package-lock.json 
~/gh/michielbdejong/solid-app-from-generator $ npm install rxjs
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN solid-app-from-generator@0.1.0 No repository field.
npm WARN solid-app-from-generator@0.1.0 No license field.

+ rxjs@6.5.2
added 2 packages from 7 contributors and audited 2 packages in 5.076s
found 0 vulnerabilities

```

**Comment by michielbdejong** [comment]
Actually, to be more precise, `npm install rxjs@6.3.3` hangs. `npm install rxjs@6.5.2` does not.

**Comment by james-martin-jd** [comment]
This issue was fixed with release 0.5.0
